src/main/java/uk/gov/justice/hmpps/casenotes/model/OffenderCaseNote.java

```diff
@@ -45,17 +46,18 @@
 @Builder(toBuilder = true)
 @EqualsAndHashCode(of = {"offenderIdentifier", "occurrenceDateTime", "locationId", "authorUsername", "caseNoteType", "noteText"})
 @ToString(of = {"id", "offenderIdentifier", "occurrenceDateTime", "locationId", "authorUsername", "caseNoteType"})
+@SQLRestriction("exists(select 1 from case_note_type ct where ct.case_note_type_id = case_note_type_id and ct.sync_to_nomis = false)")
 public class OffenderCaseNote {
 
     @Builder.Default
     @Id
     @Column(name = "OFFENDER_CASE_NOTE_ID", updatable = false, nullable = false)
     private UUID id = generateNewUuid();
 
-    @Column(nullable = false)
+    @Column(name = "occurrence_date_time", nullable = false)
```

Comment:
More for my own interest. Is there a reason to specify column name? Consistency?